.bcr/metadata.template.json

```diff
@@ -7,6 +7,9 @@
       "name": "Fabian Meumertzheim"
     }
   ],
+  "repository": [
```

Comment:
Feel free to add yourself to the list of maintainers by pushing to this branch :⁠-⁠)